src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx

```diff
@@ -160,42 +162,93 @@ function WorldWideTelescope({
     });
   }
 
-  function handleDrag(mouse) {
-    if (isDragging) {
+  function getClientXY(interaction) {
+    let position = undefined;
+    if (interaction.type === "touchstart" || interaction.type === "touchmove") {
+      if (!interaction?.touches?.[0]?.clientX || !interaction?.touches?.[0]?.clientY) {
+        return;
+      }
+      if (interaction?.touches?.length > 1) {
+        const touches = interaction.touches;
+        position = [
+          [touches[0].clientX, touches[0].clientY],
+          [touches[1].clientX, touches[1].clientY]
+        ];
+      }
+      else {
+        const touch = interaction.touches[0]; 
+        position = [touch.clientX, touch.clientY];
+      }
+    }
+    else { // mouse
+      position = [interaction.clientX, interaction.clientY];
+    }
+    if (!position) {
+      return undefined;
+    }
+    return position;
+  }
+
+  function scrollZoom(scroll) {
+    if (inverseZoom) {
+      scroll *= -1;
+    }
+    skybrowserApi.scrollOverBrowser(browserId, scroll);
+    skybrowserApi.stopAnimations(browserId);
+  }
+
+  function handleDrag(interaction) {
```

Comment:
Maybe this is nitpicky, but as this is an event handler function, I would prefer to call the `interaction` variable `event` instead. To make it clear that it is actually an object of the javascript event type :) The same goes for all the `interaction` variables in this file